examples/use-qtxdg/CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
 project(use-qtxdg)
-cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Why did you remove it here? It should (must) just be the first statement in the main CMakeLists.txt file... and this an example of project using the xdg, isn't it?

Reply:
Good point, i saw it as part of this project only.